chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts

```diff
@@ -1,15 +1,17 @@
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import * as Object from 'bytepushers-js-obj-extensions';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'angular-app';
   public showSplashPage = true;
-
+  ngOnInit() {
+    screen.orientation.lock('portrait');
```

Comment:
Does this actually work?  I don't see screen being used anywhere.

Reply:
Nope it doesnt, I was testing  it earlier but let me remove it real quick.